### PR TITLE
Added login & Signup toggle Page

### LINKED ISSUE #105: To make an attractive toggle between login & Signup pages
## Is your feature request related to a problem? Please describe.
I can see that the login page has a very common format and yet it is not catchy .I got an idea to toggle between sign up and login section so it could be user-friendly.

## Describe the solution you'd like
I would like to create a still more attractive toggle between login and signup which could be user-friendly.

### PR DESCRIPTION
**Pull request by JMKanakadurga**
#### Describe the changes you've made
I have made an attractive Sign-up and Login page which automatically toggles between the login and Sign-up.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

#### Checklist
- [ x] I have mentioned related issue number in PR description.
- [ ] I have added screenshots and related demo links of my changes..
- [ x] The title of my pull request is a short description of the requested changes.
- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Open Source Program names(OPTIONAL: If you participated in any open-source program then please mention the program name here)

### CODE REVIEW

**Comment by QAZIMAAZARSHAD** [comment]
Deploy your website on any platform then share the website link so I can review your changes.

**Comment by JMKanakadurga** [comment]
> Deploy your website on any platform then share the website link so I can review your changes.

I have created the login page is it enough to deploy that page alone or must host the entire website?

**Comment by QAZIMAAZARSHAD** [comment]
Entire website.

**Comment by JMKanakadurga** [comment]
> Deploy your website on any platform then share the website link so I can review your changes.

https://jmkanakadurga.github.io/ApniDukan/

File: index.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 <body>
     <section>
-        <nav class="navbar navbar-expand-lg navbar-light" style="background: #2980b9;">
+        <nav class="navbar navbar-expand-lg navbar-light" style="background: linear-gradient(315deg, #045de9 0%, #09c6f9 74%);">
```
Comment (QAZIMAAZARSHAD) [inline index.html:34]: Why have you modified the navbar?

**Review by QAZIMAAZARSHAD — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (QAZIMAAZARSHAD) [reply index.html:34]: Undo the changes.

Comment (JMKanakadurga) [reply index.html:34]: I have just changed the color of the navbar.Should I undo that?

Comment (QAZIMAAZARSHAD) [reply index.html:34]: Yes, no need for any other changes.

Comment (JMKanakadurga) [reply index.html:34]: Then you can merge my PR,right?

Comment (QAZIMAAZARSHAD) [reply index.html:34]: Yes. Fix it.

**Comment by QAZIMAAZARSHAD** [comment]
You need to fix few more things.

Your login page has no validation. Anything is getting accepted.
Set proper constraints for email and password.
You can also add a show password icon in passwords.

**Comment by QAZIMAAZARSHAD** [comment]
Also, improve the background.

**Comment by satyampgt4** [comment]
@JMKanakadurga any update...?

**Pull request closed without merging**